data/basejob_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "object",
+    "properties": {
+      "jobname": {
+        "type": "string"
```

Comment:
Is there a pattern for jobname? You could add in a regex

```
"jobname": {
  "type": "string",
  "pattern":  // some regex
}
```

Reply:
This one is pretty free form... I guess maybe there are some characters that may not be allowed. I'll look more into it